cpp/log/cluster_state_controller_test.cc

```diff
@@ -501,6 +501,16 @@ TEST_F(ClusterStateControllerTest, TestGetLocalNodeState) {
 
 
 TEST_F(ClusterStateControllerTest, TestNodeHostPort) {
+  // Allow some time for our view of the node state to stabilize and then
+  // check that it has. Ideally we would wait for this to definitely happen
+  // but there seems to be no easy way to arrange this.
+  sleep(1);
+  const ClusterNodeState orig_node_state(GetNodeStateView(kNodeId1));
+  EXPECT_EQ(kNodeId1, orig_node_state.hostname());
+  EXPECT_EQ(9001, orig_node_state.log_port());
```

Comment:
You're technically correct .. but I can't do this kind of loop because GetNodeStateView will fail a CHECK if the node isn't found in the peer list yet. Similarly you're right it should probably be ASSERT_EQ for clarity but in reality it will have exited before that for the same reason.